addons/spreadsheet_dashboard/static/src/bundle/grid_cursor/grid.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module */
+
+import { getNumberOfListFormulas } from "@spreadsheet/list/list_helpers";
+import spreadsheet from "@spreadsheet/o_spreadsheet/o_spreadsheet_extended";
+import { getNumberOfPivotFormulas } from "@spreadsheet/pivot/pivot_helpers";
+import { patch } from "@web/core/utils/patch";
+
+const { Grid } = spreadsheet.components;
+
+/* Used for t-key */
+let key = 0;
+
+patch(Grid.prototype, "spreadsheet_dashboard_grid_cursor", {
+    getCellClickableStyle(coordinates) {
+        return `
+      top: ${coordinates.top}px;
+      left: ${coordinates.left}px;
+      width: ${coordinates.width}px;
+      height: ${coordinates.height}px;
+    `;
+    },
+
+    getClickableCells() {
+        const cells = [];
+        const viewport = this.env.model.getters.getActiveViewport();
+        const { offsetX, offsetY } = viewport;
+        const sheetId = this.env.model.getters.getActiveSheetId();
+        for (let col = viewport.left; col < viewport.right; col++) {
+            for (let row = viewport.top; row < viewport.bottom; row++) {
+                const cell = this.env.model.getters.getCell(sheetId, col, row);
+                if (this.isClickable(cell)) {
+                    const colDim = this.env.model.getters.getColDimensions(sheetId, col);
+                    const rowDim = this.env.model.getters.getRowDimensions(sheetId, row);
+                    cells.push({
```

Comment:
Again, following the ref, these computed dimensions might not be true if we have a frozen pane.
You could however use the getter `getVisibleRect(positionToZone(col, row))` which will give you the coordinates of the cell but where 0.0 is gridOverlay, so it should work out of the box

Do you want to support mergesd cells ? because right now you don't :)